cmd/gmnhg/main.go

```diff
@@ -111,6 +115,7 @@ const (
 	defaultPageTemplate = "single"
 	indexMdFilename     = "_index.gmi.md"
 	indexFilename       = "index.gmi"
+	rssFilename         = "rss.xml"
```

Comment:
How does a crawler discover the presence of an RSS/Atom 2.0 feed on a Gemini site? With HTML, we have  `<link rel="alternate" type="application/rss+xml" href="...somewhere.xml...">`.

I recall quite a heated discussion on the Gemini mailing list about making multiple requests for page resources between Drew DeVault and someone else on the list, where DeVault said he would ban people flooding his server with unneeded requests. But I never really looked into how they currently implement RSS discovery on Gemini (if they do at all?)

Reply:
As far as I know there is no standard to speak of for discovery. From what I can see, people generally put links to their RSS feed on their homepage if they want others to find the feed.

I think Gemini sites could use some kind of optional standard URL that describes a site and includes links to feeds. Robots.txt directives could even be folded in here.